added demo for list and type

e111077 opened the pull request:
resolves #12

rictic commented on demo/index.html:
```diff
@@ -28,6 +28,14 @@
     The <code>value</code> stored at <code>key="info"</code> is <code><meta-test></meta-test></code>.
   </div>
 
+  <div class="vertical-section vertical-section-container centered">
```
This is an existing issue, but here and in the above card, consider removing the `vertical-section-container` class.

As is,
the text is
very
vertical
and hard
to read.

If a bit poetic.

rictic commented on demo/index.html:
```diff
@@ -40,6 +48,22 @@
+      is: 'type-one',
+
+      ready: function() {
+        this.textContent = new Polymer.IronMetaQuery({type: "type1"}).list;
```
Consider calling `JSON.stringify` on the list to make it clearer to the reader that it's a list.

rictic commented on demo/index.html:
```diff
@@ -28,6 +28,14 @@
   </div>
 
+  <div class="vertical-section vertical-section-container centered">
+    <iron-meta type="type1" key="a" value="Polymer"></iron-meta>
```
Consider adding a heading mentioning that you can also get list values.

e111077 commented:
Added headers and removed vertical text; here's how it should look:

![screenshot from 2016-01-28 11 36 51](https://cloud.githubusercontent.com/assets/5981958/12656503/7be110f0-c5b3-11e5-8fc6-0f3ddf16395f.png)

rictic commented:
LGTM!
